qfieldsync/gui/package_dialog.py

```diff
@@ -70,10 +70,15 @@ def __init__(self, iface, project, offline_editing, parent=None):
         self.iface = iface
         self.offliner = QgisCoreOffliner(offline_editing=offline_editing)
         self.project = project
+        self.original_project_path = self.project.fileName()
```

Comment:
This is only for internal use, right? Can we make it a "private" property by prefixing with underscore?
```suggestion
        self._original_project_path = self.project.fileName()
```

Also, please move it next to the rest of the changes in this PR a few lines below.